Sources/Components/Dialogue/DialogueView.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - DialogueViewModel
+
+/**
+ - Note:
+    Use this protocol to update Dialogue view model
+
+ */
+
+public protocol DialogueViewModel {
+    var title: String { get }
+    var detail: String { get }
+    var link: String? { get }
+    var primaryButtonTitle: String { get }
+    var secondaryButtonTitle: String? { get }
+}
+
+// MARK: - DialogueViewDelegate
+
+/**
+ - Note:
+    Protocol to call back the Dialogue view Delegate
+
+ */
+
+public protocol DialogueViewDelegate: class {
```

Comment:
We should start using `AnyObject`, instead of `class`, for class-only protocols since `class` might be [removed in the next version of Swift](https://github.com/apple/swift-evolution/blob/master/proposals/0156-subclass-existentials.md#class-and-anyobject).

Reply:
Alright! nice :)